PubMed Central.js

```diff
@@ -20,6 +44,8 @@ function detectWeb(doc, url) {
 	if (getSearchResults(doc, true)) {
 		return "multiple";
 	}
+
+	return undefined;
```

Comment:
If we're going to keep this rule, we should just return `false` for these.

Reply:
I'm fine with that, I just tried to fix the issue without introducing unnecessary semantic differences. Since `undefined` is returned implicitly, I figured making it implicit stayed closest to the original intent as expressed in the code.

Reply:
Can I change this particular commit without introducing a 3rd commit?

Reply:
Yes. Make the change, `git add` it, and then use `git commit --fixup [hash-of-commit-you're-updating]`. Then before merging, you or whoever merges this can use `git rebase` to squash the commit automatically (with either `rebase.autosquash` set to true globally, which I would recommend, or `--autosquash`).

https://riptutorial.com/git/example/3935/autosquash--committing-code-you-want-to-squash-during-a-rebase

(In this case you can just do the above and then force-push, since this is a small PR and easy to review.)

Reply:
Done the fixup and the force-push, but that still gave me a 3rd commit.

I'm not going to complain if someone else can do the rebase. rebase still terrifies me.

Reply:
Sorry, "the above" included the rebase. It'll be OK!

First, make sure this setting is on:

```
git config --global rebase.autosquash true
```

Then run `git rebase -i 6005fa6^`. That's the first commit in this PR plus ^ (which means the commit before that one).

You should see in your editor that the commits have been reordered, with the fixup commit automatically moved to after the first one. Save and close and it should be automatically merged. (Fixup means that the commit message is automatically discarded.) Then you can force push.

Technically you don't need to use `-i` in this case since there's nothing else to be done, but I always use it, since it's nice to see what's happening.

Reply:
I think that actually worked. I'll try to remember this.